tmate/create_config.py

```diff
@@ -24,14 +23,13 @@
         pass
     config[name] = {
         "username": "{}-{}".format(args.prefix, name),
-        "password": uuid4(),
+        "password":  urandom(32).hex(),
```

Comment:
This is definetly better, but I don't know if the passwords it generates are the best we can do. What about something like the following?

```
import random
import string
''.join(random.choices(string.ascii_letters + string.punctuation, k=32))
```